salt/modules/composeng.py

```diff
@@ -0,0 +1,523 @@
+# -*- coding: utf-8 -*-
+'''
+Work with containers managed by docker-compose
+
+Docker-compose method supported
+-------------------------------
+- up
+- restart
+- stop
+- start
+- pause
+- unpause
+- kill
+- rm
+- ps
+- pull
+- build
+'''
+
+from __future__ import absolute_import
+import io
+import logging
+import sys
+import traceback
+import os
+import salt.utils
+from collections import namedtuple
+from contextlib import contextmanager
+from operator import attrgetter
```

Comment:
same with the above 3 things, we do not want to use `from` because it pollutes the loader namespace